jerry-core/jerry-port.h

```diff
@@ -139,6 +139,22 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+
+
+typedef struct jerry_job_t jerry_job_t;
+typedef void (*jerry_job_handler_t) (jerry_job_t *);
+
+struct jerry_job_t
+{
+  jerry_job_handler_t handler; /**< the handler funtion for the job */
+};
+
+void jerry_port_jobqueue_init (void);
+void jerry_port_jobqueue_enqueue (jerry_job_t *job_p);
+void jerry_port_jobqueue_run (void);
```

Comment:
I prefer to keep the handler in the jerry_job_t, because it establish a connection between the callback and job (job actually is callback's arguments) in the **jerry internal**. Otherwise, the connection only exists in the port's queue item.